cluster/okd-on-ovirt/utils.sh

```diff
@@ -303,5 +303,9 @@ function download_virtctl {
 }
 
 function run_virtctl_cmd_timeout {
-  sudo timeout 10m  bash -c "export KUBECONFIG=/tmp/kubeconfig ; until /tmp/bin/virtctl ssh root@fedora-test-vm -i /tmp/id_ssh_rsa --local-ssh -c hostname --local-ssh-opts='-o StrictHostKeyChecking=no'; do sleep 10 ; done "
+  export KUBECONFIG=/tmp/kubeconfig
+
+  #find the exported vm name (assume its the only running VM)
```

Comment:
something about this comment is confusing - the vm is not necessarily running as you look for a `vm` rather than `vmi`, right?